includes/class-wc-gateway-swedbank-pay-cc.php

```diff
@@ -4,8 +4,28 @@
 	exit;
 } // Exit if accessed directly
 
-class WC_Gateway_Swedbank_Pay_Cc extends WC_Payment_Gateway_Swedbank_Pay
-	implements WC_Payment_Gateway_Swedbank_Pay_Interface {
+use SwedbankPay\Payments\WooCommerce\Adapter;
+use SwedbankPay\Payments\WooCommerce\WC_Background_Swedbank_Pay_Queue;
+use SwedbankPay\Payments\WooCommerce\WC_Swedbank_Pay_Transactions;
+use SwedbankPay\Core\Core;
+use SwedbankPay\Core\OrderInterface;
+use SwedbankPay\Core\Log\LogLevel;
+
+class WC_Gateway_Swedbank_Pay_Cc extends WC_Payment_Gateway {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class WC_Gateway_Swedbank_Pay_Cc has 18 fields. Consider redesigning WC_Gateway_Swedbank_Pay_Cc to keep the number of fields under 15.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)